configs/constant_presets/minimal.yaml

```diff
@@ -118,9 +118,9 @@ MAX_TRANSFERS: 0
 
 # Signature domains
 # ---------------------------------------------------------------
-DOMAIN_BEACON_PROPOSER: 0
-DOMAIN_RANDAO: 1
-DOMAIN_ATTESTATION: 2
-DOMAIN_DEPOSIT: 3
-DOMAIN_VOLUNTARY_EXIT: 4
-DOMAIN_TRANSFER: 5
+DOMAIN_BEACON_PROPOSER: 0x00000000
```

Comment:
Strictly these should all be `DOMAIN_TYPE_...` (as opposed to `DOMAIN_...`)

Reply:
It would push quite some lines over the line limit. I think it's ok like this. The type tells enough.